wordpress-mobile/WordPress-iOS
# Issues/8169 site creation setup

Fixes #8169 

To test:
- With the siteCreation feature flag enabled (as it is), go to My Sites > + to create a new site. Select "Create WordPress.com site". A gray screen should appear.
- Disable the siteCreation feature flag. Go to create a new again. The existing site creation screen should appear.

Needs Review: @nheagy
